packages/axelar-wasm-std/src/operators.rs

```diff
@@ -1,18 +1,47 @@
 use cosmwasm_schema::cw_serde;
-use cosmwasm_std::{to_vec, HexBinary, Uint256};
+use cosmwasm_std::{to_binary, HexBinary, Uint256};
 
 use sha3::{Digest, Keccak256};
 
+pub type OperatorsHash = [u8; 32];
```

Comment:
Do we not have a Hash type already? There's nothing special about this "Operators" hash.

Reply:
we only had `MessageHash`, I just created a type `Hash` to replace both of those